subsys/logging/log_backend_xtensa_sim.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2019 Intel Corporation Inc.
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <stddef.h>
+#include <logging/log_backend.h>
+#include <logging/log_core.h>
+#include <logging/log_msg.h>
+#include <logging/log_output.h>
+#include <xtensa/simcall.h>
+
+#define _STDOUT_BUF_SIZE 256
```

Comment:
if there a potential RAM shortages on xtensa boards this should be moved to kconfig.